### PR TITLE
Remove StorageAccountClassic Flag

### PR DESCRIPTION
**Pull request by raveeram**
- This was an exploratory addition that we will not going forward.

---
This change is [Reviewable](https://reviewable.io/reviews/azure/acs-engine/429)

### CODE REVIEW

File: parts/kubernetesmastercustomdata.yml [hunk 1]
```diff
@@ -9,6 +9,7 @@ packages:
 
 runcmd:
 - /bin/echo DAEMON_ARGS=--name "<<<variables('masterVMNames')[copyIndex()]>>>" --initial-advertise-peer-urls "<<<variables('masterEtcdPeerURLs')[copyIndex()]>>>" --listen-peer-urls "<<<variables('masterEtcdPeerURLs')[copyIndex()]>>>" --advertise-client-urls "<<<variables('masterEtcdClientURLs')[copyIndex()]>>>" --listen-client-urls "<<<concat(variables('masterEtcdClientURLs')[copyIndex()], ',http://127.0.0.1:', variables('masterEtcdClientPort'))>>>" --initial-cluster-token "k8s-etcd-cluster" --initial-cluster "<<<variables('masterEtcdClusterStates')[div(variables('masterCount'), 2)]>>>" --initial-cluster-state "new" | tee -a /etc/default/etcd
+-
```
Comment (anhowe) [inline parts/kubernetesmastercustomdata.yml]: what does this do?

Comment (raveeram) [inline parts/kubernetesmastercustomdata.yml]: This sets up etcd for each master. Expanding on what each argument does:
--name "<<<variables('masterVMNames')[copyIndex()]>>>" (etcd master host name.ex: k8s-master-D289D577-0)

--initial-advertise-peer-urls "<<<variables('masterEtcdPeerURLs')[copyIndex()]>>>" (etcd url advertised by this node to peers. see etcd cluster admin guide for more info: https://coreos.com/etcd/docs/latest/v2/clustering.html)

--listen-peer-urls (address at which this node listens for peers. same as above in our case.)

--advertise-client-urls "<<<variables('masterEtcdClientURLs')[copyIndex()]>>>" 
--listen-client-urls "<<<concat(variables('masterEtcdClientURLs')[copyIndex()], ',http://127.0.0.1:', variables('masterEtcdClientPort'))>>>"
(These are addresses at which clients can connect to this cluster. Clients like etcdctl, api-server etc)

--initial-cluster "<<<variables('masterEtcdClusterStates')[div(variables('masterCount'), 2)]>>>"
(This is the static etcd cluster we bring up. Ex: http://10.240.255.5:2379, http://10.240.255.6:2379, http://10.240.255.7:2379)

Comment (raveeram) [inline parts/kubernetesmastercustomdata.yml]: Thanks, that was a typo. Updated.

**Comment by raveeram** [comment]
@acs-bot test this please

**Pull request merged**